scripts/travis/release/release-docker.sh

```diff
@@ -3,24 +3,42 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 
-if [[ "${TRAVIS_EVENT_TYPE}" == "push" ]];
+if [[ $TRAVIS_EVENT_TYPE == "pull_request" ]];
+then
+    COMMIT_MESSAGE=$(git log -2 --pretty=%B)
+fi;
+
+echo "ℹ️ Check Docker Image release for $COMMIT_MESSAGE type $TRAVIS_EVENT_TYPE on branch $TRAVIS_PULL_REQUEST_BRANCH"
+
+if [[ $TRAVIS_EVENT_TYPE == "push" || ( $TRAVIS_EVENT_TYPE == "pull_request" && $COMMIT_MESSAGE == *"[create docker image]"* )]];
```

Comment:
Although this one works here perfectly, if you would like to free this script from CI related logic, you can either use the following conditions in the travis config file:

```yml
jobs:
  include:
    - stage: "Whatever stage"
      name: "Whatever job"
      if: type = push OR commit_message =~ *\[create docker image\]* # etc, etc... All the conditions you need
```

Or something like this, I don't remember the correct syntax, but it is somewhere in their docs.

Reply:
Don't know about grouping multiple conditions with parentheses though, like yours, but I think it is possible in the config file.